src/jquery.multiselect.js

```diff
@@ -332,21 +400,27 @@
       this.element.children().each( function() {
         var elem = this;
 
-        if (elem.tagName === 'OPTGROUP') {
+        if (elem.tagName.toUpperCase() === 'OPTGROUP') {
           var options = [];
 
           $(elem).children().each( function() {
             options.push(self._makeOption(this));
           });
 
           // Build the list section for this optgroup, complete w/ option inputs...
+          var $collapseButton = !!self.options.groupsCollapsable
+                                 ? $( document.createElement('button') )
+                                    .attr({'style': 'float:left', 'title': self.linkInfo.collapse.title})
```

Comment:
This should probably be a class instead of inline css.

Reply:
@mlh758  Ok, but I will still have the title attribute there as title cannot be set via CSS.

Reply:
That's okay.